react/Autosuggest/Autosuggest.js

```diff
@@ -68,9 +81,28 @@ export default class Autosuggest extends Component {
     );
   }
 
+  scrollOnFocus = () => {
+    // Temporary solution till we do CSS in JS
```

Comment:
My main concern is that comments like these tend to hang around forever and age really badly, particularly if plans change.